docs/install/compose-mlrun.yaml

```diff
@@ -0,0 +1,45 @@
+services:
```

Comment:
Reading https://stackoverflow.com/questions/61488222/how-to-name-docker-compose-files
I suggest that this file will be the default, therefore let's name it `docker-compose.yml` and the second file let's name `docker-compose.with-jupyter.yml`